pkg/containerutil/cp_linux.go

```diff
@@ -247,3 +275,44 @@ func mountSnapshotForContainer(ctx context.Context, client *containerd.Client, c
 	}
 	return tempDir, cleanup, nil
 }
+
+func getContainerMountInfo(ctx context.Context, con containerd.Container, containerPath string) (string, string, error) {
+	filePath := filepath.Clean(containerPath)
```

Comment:
Package name shouldn't be overwritten.

Reply:
Can you expand a little bit, unsure what you mean.

Reply:
nvm, I confused `filePath` with `filepath`.